modules/linear_boltzmann_solvers/lbs_solver/lbs_solver.cc

```diff
@@ -442,39 +432,38 @@ LBSSolver::OptionsBlock()
 
   params.SetGeneralDescription("Set options from a large list of parameters");
   params.SetDocGroup("LBSUtilities");
-
   params.AddOptionalParameter("spatial_discretization",
                               "pwld",
                               "What spatial discretization to use. Currently only `\"pwld\"` "
                               "is supported");
   params.AddOptionalParameter(
     "scattering_order", 1, "Defines the level of harmonic expansion for the scattering source.");
   params.AddOptionalParameter("max_mpi_message_size",
-                              32'768,
+                              32768,
                               "The maximum MPI message size used during sweep initialization.");
   params.AddOptionalParameter(
     "read_restart_data", false, "Flag indicating whether restart data is to be read.");
+  params.AddOptionalParameter("read_restart_directory_name",
+                              opensn::input_path.stem().string() + "_restart",
+                              "Directory name to use when reading restart data.");
+  params.AddOptionalParameter("read_restart_file_stem",
+                              opensn::input_path.stem().string(),
+                              "File stem to use when reading restart data.");
+  params.AddOptionalParameter("write_restart_directory_name",
+                              opensn::input_path.stem().string() + "_restart",
+                              "Directory name to use when writing restart data.");
   params.AddOptionalParameter(
-    "read_restart_folder_name", "YRestart", "Folder name to use when reading restart data.");
-  params.AddOptionalParameter(
-    "read_restart_file_base", "restart", "File base name to use when reading restart data.");
-  params.AddOptionalParameter(
-    "write_restart_data", false, "Flag indicating whether restart data is to be written.");
-  params.AddOptionalParameter(
-    "write_restart_folder_name", "YRestart", "Folder name to use when writing restart data.");
-  params.AddOptionalParameter(
-    "write_restart_file_base", "restart", "File base name to use when writing restart data.");
-  params.AddOptionalParameter(
-    "write_restart_interval",
-    30.0,
-    "Interval at which restart data is to be written. Currently not implemented.");
+    "write_restart_file_stem", "restart", "File stem to use when writing restart data.");
```

Comment:
This could be encapsulated into a single variable called something like `restart_dump_path` which could have the same form as `read_restart_dump` above.